scripts/plots/cam/exevs_refs_grid2obs_ctc_plots.sh

```diff
@@ -0,0 +1,422 @@
+	 echo "    >$restart/run_${stats}.${score_type}.${lead}.${VAR}.${dom}.${FCST_LEVEL_value}.${fcst_valid_hour}.completed" >> run_${stats}.${score_type}.${lead}.${VAR}.${dom}.${FCST_LEVEL_value}.${fcst_valid_hour}.sh
+	 echo "  fi" >> run_${stats}.${score_type}.${lead}.${VAR}.${dom}.${FCST_LEVEL_value}.${fcst_valid_hour}.sh
+	 echo "done" >> run_${stats}.${score_type}.${lead}.${VAR}.${dom}.${FCST_LEVEL_value}.${fcst_valid_hour}.sh
+
+         chmod +x  run_${stats}.${score_type}.${lead}.${VAR}.${dom}.${FCST_LEVEL_value}.${fcst_valid_hour}.sh 
+         echo "${DATA}/run_${stats}.${score_type}.${lead}.${VAR}.${dom}.${FCST_LEVEL_value}.${fcst_valid_hour}.sh" >> run_all_poe.sh
+
+      else
+	 #Restart from png files of previous runs
+	 for domain in $subregions ; do
+          if [ -s $restart/${score_type}_regional_${domain}_valid_${fcst_valid_hour}z_${new_var}_*.png ] ; then
+	   cp $restart/${score_type}_regional_${domain}_valid_${fcst_valid_hour}z_${new_var}_*.png ${plot_dir}/.
+	  fi
+	 done
+      fi
+
+     done #end of FCST_LEVEL_value
+
+    done #end of VAR
+
+  done #end of fcst_lead
+
+ done #end of dom 
+
+ done #end of score_type
+
+done #end of stats 
+
+done #end of valid 
+
+chmod +x run_all_poe.sh
+
+
+#***************************************************************************
+# Run the POE script in parallel or in sequence order to generate png files
+#**************************************************************************
+if [ $run_mpi = yes ] ; then
+   mpiexec -np 820 -ppn 82 --cpu-bind verbose,depth cfp ${DATA}/run_all_poe.sh
+else
+  ${DATA}/run_all_poe.sh
+fi
+export err=$?; err_chk
+
+#**************************************************
+# Change plot file names to meet the EVS standard
+#**************************************************
+cd $plot_dir
+
+for valid in 00z 03z 06z 09z 12z 15z 18z 21z ; do
+
+ for domain in conus conus_east conus_west conus_south conus_central alaska  appalachia cplains deepsouth greatbasin greatlakes mezquital midatlantic northatlantic nolains nrockies pacificnw pacificsw prairie southeast southwest splains nplains srockies ; do
+
+ if [ $domain = alaska ] ; then
+    new_domain=$domain
+ else
+    new_domain=buk_${domain}
+ fi
+
+ for var in vis hgtcldceil tcdc cape mlcape; do
+  if [ $var = vis ] ; then
+    var_new=$var
+    level=l0
+  elif [ $var = hgtcldceil ] ; then
+    var_new=ceiling
+    level=l0
+  elif [ $var = tcdc ] ; then
+    var_new=cloud
+    level=l0
+  elif [ $var = cape ] ; then
+    var_new=cape
+    level=l0
+  elif [ $var = mlcape ] ; then
+    var_new=mlcape
+    level=ml
+  fi
+
+  if [ -s performance_diagram_regional_${domain}_valid_${valid}_${var}_*.png ] ; then
+    mv performance_diagram_regional_${domain}_valid_${valid}_${var}_*.png evs.refs.ctc.${var_new}_${level}.last${past_days}days.perfdiag_valid_${valid}.${new_domain}.png
```

Comment:
The source file for this `mv` has a `*` in it (`performance_diagram_regional_${domain}_valid_${valid}_${var}_*.png`) but the destination/renaming does not `evs.refs.ctc.${var_new}_${level}.last${past_days}days.perfdiag_valid_${valid}.${new_domain}.png`. This seem a little dangerous to me. Is it just 1 performance diagram being renamed? If so, can we use the full plot name instead of `*`. If there is more than 1, then there are some edits needed to make sure they are all not being renamed to the same name.